src/System.IO.Ports/tests/Legacy/SerialPort/USB_Serial_Regression.cs

```diff
@@ -0,0 +1,89 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+// This is a regression test for Dev10 #511456, DevDiv Bugs #186131, and DevDiv Bugs #191753.
+// Unplugged USB virtual serial port causes SerialPort object to crash application in finalizer thread.
+//
+// For this manual test, you will need a USB Serial adapater, probably one like this:
```

Comment:
Yes, we have other cases of manual tests floating around other projects too (Console comes to mind). In this case, adding a skip to the attribute that you have to manually remove.